hvsock.go

```diff
@@ -7,14 +7,14 @@ import (
 	"io"
 	"net"
 	"os"
-	"syscall"
 	"time"
 	"unsafe"
 
 	"github.com/Microsoft/go-winio/pkg/guid"
+	"golang.org/x/sys/windows"
 )
 
-//sys bind(s syscall.Handle, name unsafe.Pointer, namelen int32) (err error) [failretval==socketError] = ws2_32.bind
+//sys bind(s windows.Handle, name unsafe.Pointer, namelen int32) (err error) [failretval==socketError] = ws2_32.bind
```

Comment:
I was super-disappointed that [`windows.Sockaddr` is not implementable outside `x/sys/windows`](https://github.com/golang/sys/blob/22da62e12c0cd9c1da93581e1113ca4d82a5be14/windows/syscall_windows.go#L819-L821). On the other hand, it has [a buffer that's only 116 bytes](https://github.com/golang/sys/blob/22da62e12c0cd9c1da93581e1113ca4d82a5be14/windows/syscall_windows.go#L814-L817), so we couldn't fit a `HvsockAddr` in there anyway right now. I guess it's a future problem to add `AF_HYPERV` and `HvsockAddr` to `x/sys/windows` directly.